cfdm/mixin/fielddomain.py

```diff
@@ -1,6 +1,9 @@
 import logging
 import re
 
+import numpy as np
```

Comment:
Flake8 is saying:

```console
cfdm/mixin/fielddomain.py:4:1: F401 'numpy as np' imported but unused
```

(and I've double checked it is indeed so) so this can be removed (guessing it was left over from dev work).

Reply:
Indeed it was. Fixed by https://github.com/NCAS-CMS/cfdm/pull/203/commits/8e5450c02d6b8093a6de1da6144f89c636a0d4cb